starter/script.js

```diff
@@ -1 +1,124 @@
+      'Running w the Bulls'
+      ] //(SWITCH)
+      ,
+    prompt42HowLongBasque: [
+      'How many days will you be there? (W)eek, (F)ortnight?',
+        'W', 'buy cheap camisas & espadrilles',
+        'F', 'dont sweat it, they’ll out-ripe you anyway'
+    ],
+    prompt50WhichCulturalEventsCastilian: [
+      'What cultural event do you want to see?',
+        'Bullfighter gets gored',
+        'Andalusion Dresage Throwdown'
+      ] //(SWITCH)
+      ,
+    prompt52HowLongCastilian: [
+      'How many days will you be there? (W)eek, (F)ortnight?',
+        'W', 'do your laundry in Ebro River',
+        'F', 'use 11 liters of deodorant'
+    ],
+    prompt50WhichCitySpeakCastilianWithLisp: [
+      'Do you speak Castilian?',
+        'Y',
+          'say hello to charming  Barcelona',
+      'Are you comf consorting w locals',
+        'Y',
+          'win Catalan friends, influence Catalan people',
+        'N',
+          'PromptTakeBusTour?',
+            'Y',
+              'all aboard',
+            'N',
+              'drink alone in your hotel room '
+    ],
+    prompt60WhichCulturalEventsCatalan: [
+      'What cultural event do you want to see?',
+        'Bullfighter gets gored',
+        'Priest drinks too much, starts Monastery brawl',
+        'Guerra Civil'
+    ] //(SWITCH)
+  };
+        // ARE YOU THE UGLY AMERICAN
+        // var promptUglyAmericanVar = prompt(`(${adventure.prompt20UglyAmericanStatus[2]})`);
+        var promptUglyAmericanVar = prompt(`(${adventure.prompt20UglyAmericanStatus[0]})`);
+  // console.log(promptUglyAmericanVar);
+  switch (promptUglyAmericanVar) {
+    case 'Y':
+      alert(`(${adventure.prompt20UglyAmericanStatus[2]})`);
+      break;
+    case 'N':
+      alert(`(${adventure.prompt20UglyAmericanStatus[4]})`);
+      break;
+  }
+
+          // WHICH DIALECT DO YOU SPEAK
+  var prompt30DialectSpokenVar = toUpperCase(prompt(alert(`(${adventure.prompt30DialectSpoken[0]})`)));
+
+  console.log(prompt30DialectSpokenVar);
+
+  for (let i = 0; index < adventure.prompt30DialectSpoken.length; ++index){
+    console.log('x');
+  switch (adventure.prompt30DialectSpoken[i]) {
+      case 'BAS':
+          alert(`(${adventure.prompt30DialectSpoken[1]})`);
+          break;
+      case 'CAS':
+          alert(`(${adventure.prompt30DialectSpoken[3]})`);
+          break;
+      case 'CAT':
+          alert(`(${adventure.prompt30DialectSpoken[5]})`);
+          break;
+      case 'NADA':
+          alert(`(${adventure.prompt30DialectSpoken[7]})`);
+          break;
+        }
+  }
+}
+
+
+  var varAnotherTripYn = prompt('plan another trip (Y|N)?');
+
+  switch (varAnotherTripYn) {
+    case 'Y':
+      anotherTripVarText = promptLetsStart + '\n' + prompt11SpyNameVar;
+      // alert(anotherTripVarText);
+      break;
+    case 'N':
+      text = 'Stay at home, drink alone!';
+      break;
+  // for each(var x in adventure) {
+  //   console.log(adventure.prompt20UglyAmericanStatus[x]);
+  // }
+}
```

Comment:
Your architectural approach is ambitious, but it might be simpler to debug starting out with a simpler program architecture. It hit an error at line 87 (toUpperCase is a method, ex `someString.toUpperCase()`). Great thought and effort overall, though. Start simple!